conf/test.config

```diff
@@ -22,10 +22,10 @@ params {
     gene_counts = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/merged_gene_counts.txt'
     model = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/design.txt'
     genelist = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/requested_genes.txt'
-    logFCthreshold = 1
+    logFC_threshold = 1
     //report_options = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/report_options.yml'
     project_summary = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/summary.tsv'
-    versions = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/software_versions.csv'
+    software_versions = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/software_versions.csv'
```

Comment:
Still a csv file here. Please make sure you either adapt the docu for users to know how it has to look like, or allow a yml as input

Reply:
For now, I will not change the versions stuff. I just opened an issue so that I don't forget to have another look at this in a separate PR